packages/shared-helpers/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "shared-helpers",
```

Comment:
If it _is_ going to be a public package that we ship I think we should place it under the `@patternfly` npm org, and probably give it a more descriptive name like `shared-codemod-helpers`.

Reply:
So this is valid given your comment above